Tools/pixi-packages/asan/recipe.yaml

```
@@ -0,0 +1,65 @@
+context:
+  # XXX: keep up to date
```

Re: https://discuss.python.org/t/cryptic-sequences-in-comments/38694

Can we replace all the "XXX" with "TODO" or something else? Maybe we don't even need it here:

```suggestion
  # Keep up to date
```

If we do merge this, we have code in https://github.com/python/release-tools which is run during the first beta, when `main` becomes the next version, that updates a bunch of versions in the repo. And we can later update that to also update these versions.

---

https://rattler-build.prefix.dev/latest/experimental_features/#load_from_filefile_path may remove the need for this

---

Is there a YAML, TOML, or JSON file that contains the version @hugovk ? If not, could we add one?

---

There isn't. Yes we can add one, it removes duplication here, and will be easier to programmatically update such a file.

---

any preference for format / location / name?

---

Sorry, I missed this because it was marked as resolved but I don't think it has been :)

Format: TOML or JSON, we can read using the stdlib.

Location: `Tools/pixi-packages/`, or `Tools/` so other scripts can re-use it.

Name: Hmm, `python-version.[ext]` ?

---

I think it was resolved for this PR via the third paragraph of https://github.com/python/cpython/pull/142469#pullrequestreview-3563946286

---

Okay, thanks!